### DISCUSSION TITLE
How to set "First day of week" to Saturday

### DISCUSSION THREAD

**Discussion by tmcleodskab** [kimai/kimai#2741] [General]
Hello, I am new to Kimai and it features (Aug 2021).

Question: How can I set the "first day of the week" to be Saturday for the week reporting? 

Situation:
a) We would like the "users" (commercial property maintenance managers) to be able to manage their own hours, to a maximum of 40 hours per week.
b) Typically most activates required on a Saturday and Sunday are a result of emergency or unplanned situations.
c) knowing how many hours were "accumulated" on the weekend (days 1 and 2 of the week) would allow them plan/coordinate  activities for the rest of the week with the last be being Friday.

I have looked through various posts (#2483, #2475, #2459) but have not found a solution to this specific request.  Any direction would be much appreciated.

**Comment by kevinpapst** [comment]
That's impossible (by now), as there is no such configuration.

**Reply by tmcleodskab** [reply]
Thanks for the quick response.
